### PR TITLE
#Trials- project

### PR DESCRIPTION
**Pull request by hetrox8**
This is a  simple webpage constructed using html5 and styled using css . The mian features are the image of  the autor

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 <div id="title">
 
 <h1>Dr. Norman Borlaug</h1>
-<p>Norman Ernest Borlaug was an American agronomist who led initiatives worldwide that contributed to the extensive increases in agricultural production termed the Green Revolution</p>
+<p>Norman Ernest Borlaug is an American agronomist who led initiatives worldwide that contributed to the extensive increases in agricultural production termed the Green Revolution</p>
 
 <div id="img-div">
```
Comment (hetrox8) [inline index.html:22]: I can see there's and empty div. kinly don't leave empty element . i believe you intended to use the div to add the below image. empty div make an impression of unclear code

File: index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
 <div id="title">
 
 <h1>Dr. Norman Borlaug</h1>
```
Comment (hetrox8) [inline index.html:19]: Kinldy include a header tag  for easy navigation purposes. you can include the H1 tag in the header

**Review by hetrox8 — COMMENTED** [review]
##Required  Changes 
kindly check on my comments and work on them

**Review by hetrox8 — COMMENTED** [review]
##Required  Changes 
kindly check on my comments and work on them

**Review by koachcode — APPROVED** [review]
Hi @hetrox8 , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Good use of Github inline comments 💯
- You are polite and kind in your suggestions  💯 


well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**